metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -298,6 +308,54 @@ def make_tableau_client(self, site: str) -> Server:
             ) from e
 
 
+class AccessRolesIngestionConfig(ConfigModel):
+    enable_workbooks: bool = Field(
+        default=True,
+        description="Whether or not to enable access role ingestion for workbooks. "
+        "Default: True",
+    )
+
+    group_name_pattern: AllowDenyPattern = Field(
+        default=AllowDenyPattern.allow_all(),
+        description="Filter for Tableau access permission names. "
+        "By default, all permissions will be added as roles. "
+        "You can both allow and deny permissions based on their name using their name, or a Regex pattern. "
+        "Deny patterns always take precedence over allow patterns. ",
+    )
+
+    role_prefix: str = Field(
+        default="",
+        description="Prefix to append to the group name when generating the role name used in the request_url.",
+    )
+
+    group_substring_start: Optional[int] = Field(
+        default=None,
+        description="Use this property if you only need a substring of the group in Tableau to create the role used in the request_url.",
+    )
+
+    group_substring_end: Optional[int] = Field(
+        default=None,
+        description="Use this property if you only need a substring of the group in Tableau to create the role used in the request_url.",
+    )
+
+    role_description: str = Field(
+        default="",
+        description="Description of the role that will be displayed in the Access Management tab of Datahub.",
+    )
+
+    displayed_capabilities: Optional[List[str]] = Field(
+        default=None,
+        description="Description of the role that will be displayed in the Access Management tab of Datahub.",
```

Comment:
duplicate description for fields `role_description` and `displayed_capabilities`